### ISSUE TITLE
VS2019 Can't open projects

### ISSUE THREAD

**Issue by sevek** [msawczyn/EFDesigner#272]
Hi

I downloaded the zip and opened the solution but most of the projects wouldn't open.
I couldn't find any references to the ProjectGuid's used so am unable to rectify the problem.

Could some kind soul point me in the right direction.

Cheers
Geoff

**Comment by msawczyn** [comment]
`but most of the projects wouldn't open.`

Which projects?

**Comment by sevek** [comment]
Hi Michael.

Thanks for replying.

Um, I was sure there was more but when I just reopened it only the DSL projects would not load.

DSL
    <ProjectGuid>{B77C0323-1B94-4CE4-A416-6316EBCBA81C}</ProjectGuid>

DslPackage
    <ProjectGuid>{AA22BCBD-2B02-4AD1-B8DC-19DFEBFB4436}</ProjectGuid>
    <ProjectTypeGuids>{82b43b9b-a64c-4715-b499-d71e9ca2bd60};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>

Basically, what I am wanting to do is to be able to dynamically load/create a model from a single entity definition (POCO class, perhaps attributed or with associated MAP).
The model should include the class’s immediate navigation classes.

From there I want to run custom generators.

Any help along these lines would be very much appreciated.

**Comment by sevek** [comment]
Hi Michael.

Thanks for replying.

Um, I was sure there was more but when I just reopened it only the DSL projects would not load.

DSL
    <ProjectGuid>{B77C0323-1B94-4CE4-A416-6316EBCBA81C}</ProjectGuid>

DslPackage
    <ProjectGuid>{AA22BCBD-2B02-4AD1-B8DC-19DFEBFB4436}</ProjectGuid>
    <ProjectTypeGuids>{82b43b9b-a64c-4715-b499-d71e9ca2bd60};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>

Basically, what I am wanting to do is to be able to dynamically load/create a model from a single entity definition (POCO class, perhaps attributed or with associated MAP).
The model should include the class's immediate navigation classes.

From there I want to run custom generators.

Any help along these lines would be very much appreciated.

Regards
Geoff Scott

**Comment by msawczyn** [comment]
I'm really not sure what the question is here, but if you're wanting to compile the source the only dependency is having the visual studio modeling tools installed, which is done by the visual studio installer.

**Comment by msawczyn** [comment]
Closed due to inactivity
